backend/persistent/datastores/impala.js

```diff
@@ -0,0 +1,72 @@
+import fetch from 'node-fetch';
+import {createClient} from 'node-impala';
+import {dissoc, keys, values, init, map, prepend, unnest} from 'ramda';
+
+import Logger from '../../logger';
+
+const client = createClient();
+
+export function connect(connection) {
+    return client.connect({
+        host: connection.host,
+        port: connection.port,
```

Comment:
is there a concept of auth for impala?

Reply:
Impala itself doesn't have a auth system, but the hadoop infrastructure in general has authentication ( authentication, **not** authorization) via `kerberos` . imho it is a special use case, because not everybody having hadoop infrastructure deploys kerberos -- moreover it is only an authentication mechanism (i.e. it recognizes the user who logged into hadoop services, not prevent -- or authorize them)

Reply:
Impala also authenticates using LDAP. Mentioning it here as an FYI, but we may not need to implement it unless someone specifically asks for it.